src/client/scripts/views/SignedIn/DiscoverView.js

```diff
@@ -109,25 +109,27 @@ export class DiscoverView {
 
         // user to display
         const user = await getUserById(id);
-
```

Comment:
Ignore my earlier comments. I caught you up with main. You did something wrong in your error checking, now nothing shows up in Discover or Matches.

Reply:
It's currently set to pull from the database rather than the mock data, which I'm guessing doesn't have information pulled in yet. I can check in on this once the database is ready for use.